odoo/models.py

```diff
@@ -3919,18 +3916,10 @@ def _create(self, data_list):
         other_fields = set()            # non-column fields
         translated_fields = set()       # translated fields
 
-        # column names, formats and values (for common fields)
-        columns0 = [('id', "nextval(%s)", self._sequence)]
-        if self._log_access:
-            columns0.append(('create_uid', "%s", self._uid))
-            columns0.append(('create_date', "%s", AsIs("(now() at time zone 'UTC')")))
-            columns0.append(('write_uid', "%s", self._uid))
-            columns0.append(('write_date', "%s", AsIs("(now() at time zone 'UTC')")))
-
         for data in data_list:
             # determine column values
             stored = data['stored']
-            columns = [column for column in columns0 if column[0] not in stored]
+            columns = [('id', "nextval(%s)", self._sequence)]
```

Comment:
What is nextval ?

Reply:
It is to consume one value in a sequence, thanks google :)

Reply:
This is a PostgreSQL function: it increments a sequence and returns its value.

Reply:
This is used to tell PG: "insert a new line with the next 'id' in the sequence".